### PR TITLE
LiT v0.7.0 LND v0.15.0 and js-migrations

### PR DESCRIPTION
**Pull request by cryptodread**
- [x] #3 
- [x] #2 

This also takes care of the dependency issue on LND.

### CODE REVIEW

File: manifest.yaml [hunk 1]
```diff
@@ -72,15 +75,15 @@ interfaces:
       - http
 dependencies:
   lnd:
-    version: ">=0.13.3 <0.15.0"
+    version: ">=0.14.3 <0.16.0"
```
Comment (chrisguida) [inline manifest.yaml:78]: What was the reason for this lower version bound change?

Comment (cryptodread) [reply manifest.yaml:78]: LiT requires it as a minimum version for LND as of v0.7.0.

https://github.com/lightninglabs/lightning-terminal/releases/tag/v0.7.0-alpha
"NOTE that the minimum version of lnd that can be used in --lnd-mode=remote was bumped to v0.14.3-beta."

Comment (chrisguida) [reply manifest.yaml:78]: Excellent, thanks!

File: docker_entrypoint.sh [hunk 2]
```diff
@@ -22,7 +22,7 @@ fi
 echo "Configuring LiT..."
 ### There seems to be a bug in the upstream repo, requiring lit.macaroon to be present before it is generated. 
 ### Workaround is making the empty file and letting the setup process remove it to generate a new one.
-mkdir -p ~/.lit/mainnet && touch ~/.lit/mainnet/lit.macaroon
+mkdir -p ~/.lit/mainnet
```
Comment (chrisguida) [inline docker_entrypoint.sh:25]: What's the reason for this change? Does the comment above explaining why it's here need to be updated?

Comment (cryptodread) [reply docker_entrypoint.sh:25]: yes, the comment above explains it, however I am open to finding a better solution as this is a hack to get it to work after updating.

File: scripts/services/migrations.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+          { version: "0.7.0", type: "up" },
+        ),
+        down: compat.migrations.updateConfig(
+          (config) => {
+            return config;
+          },
+          true,
+          { version: "0.7.0", type: "down" },
+        ),
+      },
```
Comment (dr-bonez) [inline scripts/services/migrations.ts:21]: if there's no migration necessary you can just leave this empty

**Review by dr-bonez — APPROVED** [review]

File: docker_entrypoint.sh (hunk 2 above)
Comment (chrisguida) [reply docker_entrypoint.sh:25]: The comment doesn't create the file anymore though... so it's wrong now, correct?

Comment (cryptodread) [reply docker_entrypoint.sh:25]: correct. I've adjusted to comment to reflect that, it only references the folder that needs to be created.

File: scripts/services/migrations.ts (hunk 3 above)
Comment (cryptodread) [reply scripts/services/migrations.ts]: I removed the unused code.

File: scripts/deps.ts [hunk 4]
```diff
@@ -1 +1,2 @@
-export * from "https://deno.land/x/embassyd_sdk@v0.3.1.0.3/mod.ts";
\ No newline at end of file
+export * from "https://deno.land/x/embassyd_sdk@v0.3.1.1.1/mod.ts";
+export * from "https://deno.land/x/embassyd_sdk@v0.3.1.0.12/emvar-lite/mod.ts";
```
Comment (chrisguida) [inline scripts/deps.ts:2]:
```suggestion
export * from "https://deno.land/x/embassyd_sdk@v0.3.1.0.12/emvar-lite/mod.ts";

```

File: scripts/embassy.ts [hunk 5]
```diff
@@ -1,4 +1,5 @@
 export { setConfig } from "./services/setConfig.ts";
 export { properties } from "./services/properties.ts";
 export { getConfig } from "./services/getConfig.ts";
-export { dependencies } from "./services/dependencies.ts";
\ No newline at end of file
+export { dependencies } from "./services/dependencies.ts";
+export { migration } from "./services/migrations.ts";
```
Comment (chrisguida) [inline scripts/embassy.ts:5]:
```suggestion
export { migration } from "./services/migrations.ts";

```

File: scripts/services/migrations.ts [hunk 6]
```diff
@@ -0,0 +1,4 @@
+import { compat, types as T } from "../deps.ts";
+
+export const migration: T.ExpectedExports.migration = compat.migrations
+  .fromMapping( {}, "0.7.0" );
```
Comment (chrisguida) [inline scripts/services/migrations.ts:4]:
```suggestion
  .fromMapping( {}, "0.7.0" );

```

File: scripts/services/setConfig.ts [hunk 7]
```diff
@@ -1,4 +1,4 @@
 import {
   compat,
 } from "../deps.ts";
-export const setConfig = compat.setConfig;
\ No newline at end of file
+export const setConfig = compat.setConfig;
```
Comment (chrisguida) [inline scripts/services/setConfig.ts:4]: ✅

**Review by chrisguida — APPROVED** [review]
Some missing newlines, otherwise looks good
